internal/host/common.go

```diff
@@ -19,17 +19,18 @@ import (
 )
 
 const (
-	statusInfoDisconnected               = "Host keepalive timeout"
-	statusInfoDisabled                   = "Host is disabled"
-	statusInfoDiscovering                = "Waiting for host hardware info"
-	statusInfoInsufficientHardware       = "Host does not pass minimum hardware requirements"
-	statusInfoPendingForInput            = "User input required"
-	statusInfoNotReadyForInstall         = "Host not ready for install"
-	statusInfoInstalling                 = "Installation in progress"
-	statusInfoResettingPendingUserAction = "Reboot the host into the installation image to complete resetting the installation"
-	statusInfoPreparingForInstallation   = "Preparing host for installation"
-	statusInfoPreparingTimedOut          = "Cluster is no longer preparing for installation"
-	statusInfoAbortingDueClusterErrors   = "Installation has been aborted due cluster errors"
+	statusInfoDisconnected               = "Host has stopped communicating with the installation service"
+	statusInfoDisabled                   = "Host was disabled by user"
+	statusInfoDiscovering                = "Waiting for host to send hardware details"
+	statusInfoInsufficientHardware       = "Host does not possess minimum hardware requirements"
+	statusInfoPendingForInput            = "Waiting for user input"
+	statusInfoNotReadyForInstall         = "Host cannot be installed due to failing validation(s)"
+	statusInfoKnown                      = "Host is ready to be installed"
+	statusInfoInstalling                 = "Installation is in progress"
+	statusInfoResettingPendingUserAction = "Host requires booting into the installation image to complete resetting the installation"
```

Comment:
```suggestion
	statusInfoResettingPendingUserAction = "Host requires booting into the discovery image to complete resetting the installation"
```

I'm not sure about this suggestion, but should we generally refer to the image as the "Discovery" ISO/Image?

Reply:
Yes, good idea, done